src/features/auth/login-form/LoginForm.tsx

```diff
@@ -2,38 +2,105 @@ import TextField from "@mui/material/TextField";
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+    const FormBoxStyles = {
+        width: isSmallScreen ? '85%' : '350px',
+        height: '480px',
+        margin: 'auto', 
+        padding: isSmallScreen ? '18px' : '40px',
+        backgroundColor:'white',
+        display: 'flex',
+        flexDirection:'column',
+        alignItemns:'center',
+        justifyContent:'center',
+        borderRadius:'8px',
+        marginBottom:'16px',
+    };
 
   return (
     <>
-      <TextField
-        value={email}
-        label="Correo electronico"
-        type="email"
-        onChange={onChangeEmail}
-        disabled={disabled}
-      />
-      <TextField
-        value={password}
-        label="Contraseña"
-        type="password"
-        onChange={onChangePassword}
-        disabled={disabled}
-      />
-      <Button
-        variant="contained"
-        color="primary"
-        type="submit"
-        disabled={disabled}
-        onClick={() => {
-          onSubmit(email, password);
-        }}
-      >
-        Iniciar sesion
-      </Button>
+
+      <Box sx={FormBoxStyles}>
+        <Box sx={ImageBoxStyles}>
+          <img src={ucabLogo} alt="UCAB Logo" style={UcabLogoStyles} />
+        </Box>
+        <Box>
+          <Typography sx={TextFieldTypography}>Correo electronico</Typography>
+          <TextField
+            value={email}
+            label=""
+            type="email"
+            onChange={onChangeEmail}
+            disabled={disabled}
+            sx={TextFieldStyles}
+            InputProps={{ sx: { borderRadius: '11px' } }}
+          />
+          <Typography sx={TextFieldTypography}>Contraseña</Typography>
+          <TextField
+            value={password}
+            label=""
+            type="password"
+            onChange={onChangePassword}
+            disabled={disabled}
+            sx={{ ...TextFieldStyles, marginBottom: '26px' }}
+            InputProps={{ sx: { borderRadius: '11px' } }}
+          />
+          <Box>
+            <Button
+              variant="contained"
+              color="primary"
+              type="submit"
+              disabled={disabled}
+              onClick={() => {
+                onSubmit(email, password);
+              }}
+              sx={ButtonStyles}
+            >
+              Iniciar Sesión
+            </Button>
+          </Box>
+          <Box sx={FormBottomStyles}>
+            <Box sx={FormBottonStyles1}>
+              <Check/> 
+              <Typography sx={FormBottomTypographyStyles}>Permanecer conectado</Typography>
+            </Box>
+            <Box sx={ForgotPaswordStyles}>
+              <Link href="#" rel="noopener noreferrer">
+                ¿Olvidó la contraseña?
+              </Link>
+            </Box>
+          </Box>
+        </Box>
+      </Box>
+      <Box sx={UnderFormStyles}> 
+          <Box sx={{display:'inline-block',}}>¿Aún no tienes cuenta?</Box>   &nbsp;  &nbsp;  &nbsp;
```

Comment:
```suggestion
          <Box sx={{display:'inline-block',}}>¿Aún no tienes cuenta?</Box>&nbsp;&nbsp;&nbsp;
```
Aunque idealmente esto deberia hacerse con flexbox, margenes o algo un poquito mas elegante...
```suggestion
          <Box sx={{display:'inline-block',}}>¿Aún no tienes cuenta?</Box>
```

Reply:
Definitivamente, hace falta un space-between, debe estar separado de extremo a extremo no solo un par de espacios, esta es una forma muy mala de intentar replicar esto
![image](https://github.com/EquestriaReapers/ER-frontend/assets/22270579/0f80cacf-5704-420e-b12d-85ebbe4b1937)